build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
+        {
+            "kind": "PerBuildPreRelease", 
```

Comment:
> Yes. I think this is our long term location.

Let me clarify a bit. I think eventually we will be publishing more than 1 RTM package. Once we switch to RTM we will only publish RTM for htat particular channel. How often we do it is up for debate.